src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -80,6 +93,21 @@ def get_household_details(self, idx: pd.Index) -> pd.DataFrame:
         household_details = household_details.astype(HOUSEHOLD_DETAILS_DTYPES)
         return household_details
 
+    ########################
+    # Event-driven methods #
+    ########################
+
+    def on_initialize_simulants(self, pop_data: SimulantData) -> None:
+        "Sample the GQ household states/pumas on initialization"
```

Comment:
I don't see why to do this on initialization instead of in setup. Don't you already have everything you need when the GQ households are created in the first place?

Reply:
Ah, I was wondering if it had to do with that. Ok, this is fine for now.